src/daidepp/keywords/base_keywords.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, unit, province, *province_seas):
     def __str__(self):
         return (
             f"( {self.unit} ) CTO {self.province} VIA ( "
-            + " ".join(sorted(map(lambda x: str(x), self.province_seas)))
```

Comment:
my issue with this is that we're using `x` for a non-integer value. use `s` or `province` or `p` or something!

also you can just map `str`, no need for the lambda...

Reply:
`self.province_seas` is actually already a list of `str`s, so I removed the `map` entirely.